recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/BaseElement.java

```diff
@@ -149,6 +149,15 @@ public Element getChildAt(int line, int column) {
 
 	}
 
+	public Element getChildAt(int position) {
+		for (Element child : getChildren()) {
+			if (child.getStart() <= position && child.getStop() >= position) {
```

Comment:
Incredibly nit-picky, but I think `pos >= start && pos <= stop` is more immediately legible than `start <= pos && stop >= pos`.

Supporting evidence:
 - My sleep deprived brain was confused at first glance here